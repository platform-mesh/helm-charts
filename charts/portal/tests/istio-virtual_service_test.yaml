suite: virtual service
templates:
  - istio-virtual-service.yaml
values:
  - ../test-values.yaml
release:
  name: portal
tests:
  - it: virtual service match the snapshot
    set:
      global:
        istio:
          enabled: true
    asserts:
      - matchSnapshot: {}
