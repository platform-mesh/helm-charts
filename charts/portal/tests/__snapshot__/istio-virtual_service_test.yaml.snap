virtual service match the snapshot:
  1: |
    apiVersion: networking.istio.io/v1alpha3
    kind: VirtualService
    metadata:
      name: portal
    spec:
      gateways:
        - NAMESPACE/gateway
      hosts:
        - '*'
      http:
        - corsPolicy:
            allowHeaders:
              - Authorization
              - Content-Type
              - '*'
            allowMethods:
              - GET
              - POST
              - PUT
              - DELETE
          route:
            - destination:
                host: portal.NAMESPACE.svc.cluster.local
                port:
                  number: 8080
